src/Components/Server/src/Circuits/RenderCompletionException.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Web.Rendering
+{
+    internal class RenderCompletionException : Exception
```

Comment:
Where is this *caught*? Do we need this type if no one is catching it?

Reply:
This was RemoteRenderer, we have two internal exceptions RenderCompletionException and InvalidRenderAcknowledgeException. We don't catch any of them specifically, but their name would show up in the logs.

We can simply remove them, I don't feel strongly about it.

Reply:
(Removing them was something I already considered, just wasn't sure what your opinions would be)